### DISCUSSION TITLE
load  FontFamily

### DISCUSSION THREAD

**Discussion by fufufu3** [AvaloniaUI/Avalonia#7316] [Q&A]
i can't load the font that save out Assembly
`FontFamily newFontFamily = new FontFamily(new Uri("file:///C:/Users/10402/Desktop/font/"), "#ZQKHYT");`
when i using the FontFamily  report an error (
      System.InvalidOperationException:“Could not create glyph typeface for: ZQKHYT.”
)
But it run well   when i add it as AvaloniaResource then use `new FontFamily(new Uri("avares://xxxx/Res/"), "#ZQKHYT")`
how can i  do ? avalonia can't do it with the way?

**Comment by Gillibald** [comment]
The `file` scheme is currently not supported

**Reply by fufufu3** [reply]
oh thanks .Any plans for that later?

**Reply by Whiletru3** [reply]
Hello,

I also need this feature, any news about it ?
Thanks
